### PR TITLE
New aport netatalk

### PR DESCRIPTION
**Pull request by alexfoo**
Netatalk is a freely-available Open Source AFP fileserver.

### CODE REVIEW

File: testing/netatalk/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+pkgname=netatalk
+pkgver="3.1.10"
+pkgrel=0
+pkgdesc="Netatalk is a freely-available Open Source AFP fileserver."
+url="http://netatalk.sourceforge.net/"
+arch="all"
+license="GNU GPLv2"
+depends="db libgcrypt acl"
+makedepends="db-dev libgcrypt-dev acl-dev"
+#install="$pkgname.pre-install $pkgname.post-install"
```
Comment (jirutka) [inline testing/netatalk/APKBUILD]: Please do not comment-out code, just remove it.

File: testing/netatalk/netatalk.initd [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/sbin/openrc-run
+
+name=netatalk
+command="/usr/sbin/$name"
+command_args="-F $netatalk_config"
```
Comment (jirutka) [inline testing/netatalk/netatalk.initd]: Runscripts should have sane defaults if possible, i.e. don’t require config file in /etc/conf.d. Also more common name for config file is `conffile`.

File: testing/netatalk/netatalk.initd [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+command_background="yes"
+
+extra_started_commands="reload"
```
Comment (jirutka) [inline testing/netatalk/netatalk.initd]: You didn’t write the actual reload function and IIRC there’s no default reload function provided by OpenRC.

File: testing/netatalk/netatalk.initd [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+pidfile="/run/$name.pid"
+
+depend() {
+	need net
+	after firewall
+}
+
+stop() {
+	ebegin "Stopping $name"
+	start-stop-daemon -K -p /var/lock/netatalk
```
Comment (jirutka) [inline testing/netatalk/netatalk.initd]: What’s the point of this? Default stop command doesn’t work?

Comment (alexfoo) [inline testing/netatalk/netatalk.initd]: The built in stop command didn't work. I'm not familiar with openrc-stuff so I just made-it-work(tm).

File: testing/netatalk/netatalk.initd (hunk 3 above)
Comment (alexfoo) [inline testing/netatalk/netatalk.initd]: Removed

File: testing/netatalk/netatalk.initd (hunk 2 above)
Comment (alexfoo) [inline testing/netatalk/netatalk.initd]: Fixed

File: testing/netatalk/APKBUILD (hunk 1 above)
Comment (alexfoo) [inline testing/netatalk/APKBUILD]: Removed

**Comment by kaniini** [comment]
Can you please flatten all changes?  Thanks

File: testing/netatalk/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Contributor:
+# Maintainer: Alexander Rigbo <alex@dnb.nu>
+pkgname=netatalk
+pkgver="3.1.10"
+pkgrel=0
+pkgdesc="Netatalk is a freely-available Open Source AFP fileserver."
+url="http://netatalk.sourceforge.net/"
+arch="all"
+license="GNU GPLv2"
+depends="db libgcrypt acl"
```
Comment (jirutka) [inline testing/netatalk/APKBUILD]: These dependencies are autodetected via dynamic linking, so it’s not needed (nor desirable) to list them here.

**Comment by jirutka** [comment]
I’ve fixed some other issues (bundled dependencies, runscript, …) in next commits.

**Comment by jirutka** [comment]
@alexfoo Could you please verify that this package really works and let me know? I’d like to move it into the community repo.

**Comment by vi7** [comment]
@jirutka `netatalk-3.1.10-r1` installed and launched on `alpine:edge` Docker container. Tested with OS X 10.10.5 - everything is working fine. When it might be available through the community repo?

**Comment by vi7** [comment]
@jirutka any updates on moving the netatalk to the community repo?

**Comment by jirutka** [comment]
Sorry, I forgot about it. I’ve moved it to the community a while ago.

**Comment by vi7** [comment]
@jirutka thank you!

**Pull request closed without merging**